[2022-11-02T11:51:16.216+0700] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: my_python_op_dag.get_time manual__2022-11-02T04:51:14.469753+00:00 [queued]>
[2022-11-02T11:51:16.222+0700] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: my_python_op_dag.get_time manual__2022-11-02T04:51:14.469753+00:00 [queued]>
[2022-11-02T11:51:16.222+0700] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-11-02T11:51:16.222+0700] {taskinstance.py:1363} INFO - Starting attempt 1 of 6
[2022-11-02T11:51:16.222+0700] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-11-02T11:51:16.232+0700] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): get_time> on 2022-11-02 04:51:14.469753+00:00
[2022-11-02T11:51:16.234+0700] {standard_task_runner.py:55} INFO - Started process 41788 to run task
[2022-11-02T11:51:16.236+0700] {standard_task_runner.py:82} INFO - Running: ['airflow', 'tasks', 'run', 'my_python_op_dag', 'get_time', 'manual__2022-11-02T04:51:14.469753+00:00', '--job-id', '31', '--raw', '--subdir', 'DAGS_FOLDER/python_op_dag.py', '--cfg-path', '/tmp/tmp5n4p0n60']
[2022-11-02T11:51:16.237+0700] {standard_task_runner.py:83} INFO - Job 31: Subtask get_time
[2022-11-02T11:51:16.271+0700] {task_command.py:376} INFO - Running <TaskInstance: my_python_op_dag.get_time manual__2022-11-02T04:51:14.469753+00:00 [running]> on host ginraca-VirtualBox
[2022-11-02T11:51:16.310+0700] {taskinstance.py:1590} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=admin
AIRFLOW_CTX_DAG_ID=my_python_op_dag
AIRFLOW_CTX_TASK_ID=get_time
AIRFLOW_CTX_EXECUTION_DATE=2022-11-02T04:51:14.469753+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-11-02T04:51:14.469753+00:00
[2022-11-02T11:51:16.318+0700] {python.py:177} INFO - Done. Returned value was: None
[2022-11-02T11:51:16.320+0700] {taskinstance.py:1401} INFO - Marking task as SUCCESS. dag_id=my_python_op_dag, task_id=get_time, execution_date=20221102T045114, start_date=20221102T045116, end_date=20221102T045116
[2022-11-02T11:51:16.332+0700] {local_task_job.py:164} INFO - Task exited with return code 0
[2022-11-02T11:51:16.343+0700] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
